spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/controller/AboutControllerTests.java

```diff
@@ -398,6 +400,8 @@ public void testAbout() throws Exception {
 					.andExpect(jsonPath("$.runtimeEnvironment.appDeployer.deployerName", is("skipper server")))
 					.andExpect(jsonPath("$.monitoringDashboardInfo.url", is("http://localhost:3001")))
 					.andExpect(jsonPath("$.monitoringDashboardInfo.dashboardType", is("GRAFANA")))
+					.andExpect(jsonPath("$.gitAndBuildInfo.git.commit").exists() )
```

Comment:
Does `exists()` fail if the value is empty on the field? Maybe instead we can use a `hasText` or `notEmpty` matcher. Wdyt?

Reply:
yes... better. Done